Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzill41559.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+using Xamarin.Forms;
+using Xamarin.Forms.CustomAttributes;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Issue(IssueTracker.Bugzilla, 41559, "In iOS Entry.keyboard=Keyboard.Numeric does not have a done button")]
+	public class Bugzill41559 : TestContentPage
+	{
+		protected override void Init()
+		{
+			var numericEntry = new Entry { Keyboard = Keyboard.Numeric, AutomationId = "NumericEntry" };
+			Content = new StackLayout
+			{
+				Children = {
+					numericEntry
+				}
+			};
+
+			numericEntry.Focus();
+		}
+
+#if UITEST
+		[Test]
+		public void Bugzilla41559Test ()
+		{
+			RunningApp.Tap("NumericEntry");
```

Comment:
This isn't quite right, and the test doesn't actually compile. You might try doing just `RunningApp.WaitForElement(c => c.Marked("Done"));` This will look for an element with Text or AutomationId == "Done". If there is no element after a few seconds, then the test will fail. Otherwise, it will pass. 👎 until this is fixed (so we don't break the build by merging this in).